APP Direct/README.md

```diff
@@ -1,19 +1,23 @@
-# Indexing AppDirect Using the REST API Connector
+# Indexing AppDirect using the Coveo REST API connector
+This guide explains how you can use the content of the [`SourceJSONConfig.json`](SourceJSONConfig.json) file in a [REST API](https://docs.coveo.com/en/1896/) source to index products from AppDirect. Your Coveo source will use this JSON configuration to customize HTTP requests for the AppDirect v1 API and identify the specific content to extract from the responses.
 
-## Use Case
-This example shows how to add products from AppDirect.
+## Disclaimer
+When [adding a source of content](https://docs.coveo.com/en/3390/index-content/add-or-edit-a-source#add-a-source) in the [Coveo Administration Console](https://docs.coveo.com/en/1841/), Coveo may recommend, or not recommend, using a Coveo [REST API](https://docs.coveo.com/en/1896/) or [GraphQL API](https://docs.coveo.com/en/n6gh2329/) source along with the associated example JSON configuration from this library. Coveo’s recommendation depends on the extent of testing of the system example configuration in proofs of concept.
```

Comment:
> Plus, users may have ended up here thanks to the recommendations we'll implement in the UI, sure, but possibly also thanks to the docs.

I plan to tweak the connector directory article and https://docs.coveo.com/en/3390/index-content/add-or-edit-a-source#add-a-source so that our docs, source creation in the Console, and the connectivity library are well aligned.

For now, I'll leave the Advisory wording as is, but this might well evolve based on feedback we get.